packages/suite/src/views/wallet/coinmarket/exchange/offers/Offers/SelectedOffer/components/VerifyAddress/ReceiveOptions/index.tsx

```diff
@@ -64,20 +64,23 @@ type FormState = {
     extraField?: string;
 };
 
-const VerifyAddressComponent = () => {
+type Props = Pick<UseFormMethods<FormState>, 'setValue'> & {
+    selectedAccountOption: AccountSelectOption | undefined;
```

Comment:
Nitpick: `selectedAccountOption: AccountSelectOption | undefined;` could be `selectedAccountOption?: AccountSelectOption;`

Reply:
Done